src/phoenix/db/models.py

```diff
@@ -86,8 +104,8 @@ class Base(DeclarativeBase):
         }
     )
     type_annotation_map = {
-        Dict[str, Any]: JSON_,
-        List[Dict[str, Any]]: JSON_,
+        Dict[str, Any]: JsonDict,
+        List[Dict[str, Any]]: JsonList,
```

Comment:
Out of curiosity, instead of using this conversion map, can we just annotation with the `JsonDict` and `JsonList` types directly?

Reply:
Interesting thought, but i think they are not equivalent, because one refers to the (python) type of values on the instance objects, where the other refers to the type on the class object which has direct ties to the database.